### ISSUE TITLE
Train Data Corrupted for I3D ActivityNet Feature

### ISSUE THREAD

**Issue by sauradip** [Alvin-Zeng/PGCN#46]
Hi, Thanks for uploading the I3D features for ANet. 

However, i found out that the RGB features zipped (in Gcloud )  are corrupted. Can you double check the current version or upload the corrected features ? There are too many files in I3D ANet RGB features link. 

Thanks in advance

**Comment by doltonfernandes** [comment]
They are not corrupted. The main zip file has been divided into multiple files for ease of download/upload by the authors.
Use this command to merge them into one single zip file:
zip -F rgb-resize-step16-train.zip --out output.zip

**Comment by sauradip** [comment]
> They are not corrupted. The main zip file has been divided into multiple files for ease of download/upload by the authors.
> Use this command to merge them into one single zip file:
> zip -F rgb-resize-step16-train.zip --out output.zip

Thanks @doltonfernandes for the help, do you have the FLow Features ? I cannot access this from EU. Can you send me here if you have : s.nag@surrey.ac.uk

**Comment by doltonfernandes** [comment]
I haven't downloaded them. Maybe try using VPN to get access ?

**Comment by sauradip** [comment]
> I haven't downloaded them. Maybe try using VPN to get access ?

Ok Thank you. I will try. I also saw while running you need the .pkl file which is created in dataloader. That file is consuming my RAM and eventually stopped. Do you have the preprocessed pkl file for ANET ? If you have can you kindly share the link or send me via email above ?

**Comment by doltonfernandes** [comment]
Yeah even I noticed that there are some dictionaries in pgcn_dataset.py created which are huge in size. What I did as a workaround was to use [chest dictionaries](https://pypi.org/project/chest/) instead of those ones.
